xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -256,27 +256,19 @@ viewers.Comments = Class.create({
     }.bindAsEventListener(this));
   },
   /**
-   * Permalink: Display a modal popup providing the permalink.
+   * Permalink: Display a bootstrap modal providing the permalink.
    */
   addPermalinkListener : function() {
     $$(this.xcommentSelector + ' a.permalink').each(function(item) {
       item.observe('click', function(event) {
-        item.blur();
-        event.stop();
-        var permalinkBox = new XWiki.widgets.ConfirmationBox(
-        {
-          onYes : function () {
-            window.location = item.href;
-          }
-        },
-        /* Interaction parameters */
-        {
-          confirmationText: "$services.localization.render('core.viewers.comments.permalink'): <input type='text' class='full' value='" + item.href + "'/>",
-          yesButtonText: "$services.localization.render('core.viewers.comments.permalink.goto')",
-          noButtonText : "$services.localization.render('core.viewers.comments.permalink.hide')"
+        //Updating the permalink inside modal
```

Comment:
Our code style is to leave a space after ``//``.

Reply:
Also, I would take this opportunity to move this code to jQuery. I would use the ``show.bs.modal`` event instead.